Updated readme file (#1)
H2020-OpenModel/ontologies: README.md

OpenModel application ontologies
================================

> A few application ontologies have been made public available in OpenModel.

Reviewer: ```suggestion
A few application ontologies have been made publicly available in OpenModel.
```

Computation application ontology
--------------------------------

> The [computation application ontology] ontologise the terms used by the

Reviewer: ```suggestion
The [computation application ontology] ontologises the terms used by the
```

[ExecFlowAPI], which is a REST API for configuring [ExecFlow] to run
simulations on remote HPC clusters.

Success story 3 application ontology
------------------------------------
The [success story 3 application ontology] is a simple open

> application ontology describing the workflow in OpenModel [Success

Reviewer: ```suggestion
application ontology describing inputs and outputs of models to be used  in OpenModel [Success
```
